src/main/java/nl/tudelft/ewi/devhub/server/web/resources/repository/AbstractProjectResource.java

```diff
@@ -600,29 +604,56 @@ public Response showSettings(@Context HttpServletRequest request) throws IOExcep
 	@POST
 	@Path("/branches/delete")
```

Comment:
To conform REST-standards, it should have been `DELETE /branch/{branchName}`. But as `DELETE` is not supported in HTML4 forms, we have to use a `POST` alias request. I think `/branch/{branchName}` might be a little more consistent with the other requests, so it improves readability, but on the other hand, dealing with slashes in URL's is way more difficult, so we may not want to go through the hassle.